backend/src/controllers/report.js

```diff
@@ -188,7 +227,7 @@ controller.getProjectStatusSummaryReportOnRequest = async (request, reply) => {
       alignment: await model.getStrategicAlignment(projectId),
       statuses: await model.getProjectStatuses(projectId),
       lessons: await model.getLessonsLearned(projectId),
-      reportDate: reportDate,
+      report_date: reportDate,
```

Comment:
We should be using camel case like the rest of the app

Reply:
@ASpiteri-BCGov  here we probably shouldn't.. this isn't to do with js, but more the variables that the template has, since we use snake_case in the db fields, it makes sense to use it here so in the template there are not snake_case and camelCase..  This was discussed on previous pr